### ISSUE TITLE
Installing multiple adders via CLI flags doesn't work

### ISSUE THREAD

**Issue by benmccann** [svelte-add/svelte-add#420]
### Topic

svelte-add

### Version

2.1.0

### Expected behavior

`npx svelte-add --help` should say what syntax is expected and then that syntax should work. I tried using both a space and comma, but neither worked. Passing unexpected adders should error.

Even nicer would be to make the `--adder` flag unnecessary. E.g.:
```
npx svelte-add tailwind drizzle
```

### Actual behavior

`npx svelte-add --adder tailwind drizzle` installed only drizzle

`npx svelte-add --adder tailwind,drizzle` asked me what package manager to use and then didn't seem to do anything

**Comment by AdrianGonz97** [comment]
> `npx svelte-add --adder tailwind drizzle` installed only drizzle

I'm not experiencing this and it looks to be working for me 🤔 perhaps you're hitting a cached version of the package if you're not including the `@latest` tag in the command?

This is what I'm getting:
```
➜  svelte-add-testing pn dlx svelte-add@latest --adder tailwindcss drizzle                   
Packages: +33
+++++++++++++++++++++++++++++++++
Progress: resolved 33, reused 23, downloaded 10, added 33, done
svelte-add version 2.1.0

┌  Welcome to Svelte Add!
│
◇  Drizzle: Which database would you like to use?
│  MySQL
│
◇  Drizzle: Which MySQL client would you like to use?
│  mysql2
│
◇  Drizzle: Do you want to run the database locally with docker-compose?
│  No
│
◇  TailwindCSS: Do you want to use typography plugin?
│  No
│
◇  Successfully installed dependencies
│
└  You're all set!
```

and everything gets installed correctly too

---

> Even nicer would be to make the `--adder` flag unnecessary. E.g.:
> 
> ```
> npx svelte-add tailwind drizzle
> ```

this is my preference as well, though I'm not sure if there's a technical reason behind the need of the `--adder` flag

**Comment by benmccann** [comment]
> I'm not experiencing this and it looks to be working for me 🤔 perhaps you're hitting a cached version of the package if you're not including the @latest tag in the command?

Wow, that's it. But it's super weird because the first line of output is `svelte-add version 2.1.0` either way!

Here's without `@latest`:
```
$ npx svelte-add --adder tailwind drizzle
svelte-add version 2.1.0

┌  Welcome to Svelte Add!
│
◇  Which database would you like to use?
│  MySQL
│
◇  Which MySQL client would you like to use?
│  mysql2
│
◇  Do you want to run the database locally with docker-compose?
│  No
│
◇  Successfully installed dependencies
│
└  You're all set!
```

Notice that it's not prefixing the questions with `Drizzle:` like in your version. Any idea when that was added? Is it fetching any dependencies dynamically that could be affected? I'm so confused how it can be saying `version 2.1.0` but be running different code

**Comment by AdrianGonz97** [comment]
> Wow, that's it. But it's super weird because the first line of output is svelte-add version 2.1.0 either way!

woah! that is weird...

> Is it fetching any dependencies dynamically that could be affected? I'm so confused how it can be saying version 2.1.0 but be running different code

hmm, you may be right on that. the logic for that bit resides in `@svelte-add/core`, which is externalized. my guess is that a version of `@svelte-add/core` that didn't have those prefixes was cached on your machine and was used instead, but _only_ when you didn't add the `@latest` tag. how strange 🤔

**Comment by manuel3108** [comment]
> this is my preference as well, though I'm not sure if there's a technical reason behind the need of the `--adder` flag

I cannot remember one for sure. v1 had the same flag, so I did not thought much about it and just converted it to v2. Although I'm currently not sure if and how we could communicate this to [`commander`](https://www.npmjs.com/package/commander)

> Notice that it's not prefixing the questions with `Drizzle:` like in your version. Any idea when that was added? Is it fetching any dependencies dynamically that could be affected? I'm so confused how it can be saying `version 2.1.0` but be running different code

It's only adding that prefix if it's detected to be run from the `svelte-add` cli. It does this if it detects that there are multiple possible adders that can be applied.

> `npx svelte-add --adder tailwind,drizzle` asked me what package manager to use and then didn't seem to do anything

As you have found that's not the expected command.

> > Is it fetching any dependencies dynamically that could be affected? I'm so confused how it can be saying version 2.1.0 but be running different code
> 
> hmm, you may be right on that. the logic for that bit resides in `@svelte-add/core`, which is externalized. my guess is that a version of `@svelte-add/core` that didn't have those prefixes was cached on your machine and was used instead, but _only_ when you didn't add the `@latest` tag. how strange 🤔

That's really interesting!

With the changes we have done lately, we might be able to bundle `@svelte-add/x` again. At the time I removed that because it was re-downloading dependencies. But Im not to sure about that one.

**Comment by benmccann** [comment]
> Although I'm currently not sure if and how we could communicate this to [commander](https://www.npmjs.com/package/commander)

It looks like they call it variadic arguments and give this example with `...`:
```
program
  .version('0.1.0')
  .command('rmdir')
  .argument('<dirs...>')
  .action(function (dirs) {
    dirs.forEach((dir) => {
      console.log('rmdir %s', dir);
    });
  });
```

**Comment by manuel3108** [comment]
Besides the simplified usage (#429) is there any reproducible issue remaining here?

I was thinking to increase our version output in the cli to something like this
```
svelte-add version 2.1.1 (core: 2.x.x)
```

But as this appears to be a one of occurrence at this point, I don't think it matters to much right now. Additionally we recommend using `@latest` in our readmes and on the website.

**Comment by AdrianGonz97** [comment]
> But as this appears to be a one of occurrence at this point, I don't think it matters to much right now. Additionally we recommend using `@latest` in our readmes and on the website.

I wonder if there's a way to detect if they're using the `@latest` tag and possible error out (or log a warning) if it's missing? I think it would be a helpful way to tell the user "hey! this version is possibly out of date, use: `svelte-add@latest instead!"

**Comment by manuel3108** [comment]
I think `process.argv` should spit that out. Might give it a try later

**Comment by benmccann** [comment]
> Additionally we recommend using @latest in our readmes and on the website.

I don't on kit.svelte.dev or `create-svelte`. I might have to update those, but I'd really like to find out how the heck it's getting the latest version of the main CLI and old versions of core or whatever it's doing

**Comment by AdrianGonz97** [comment]
> but I'd really like to find out how the heck it's getting the latest version of the main CLI and old versions of core or whatever it's doing

I'm starting to think this is an `npx` bug. Looking at the `package.json` for `svelte-add@2.1.0`: https://www.npmjs.com/package/svelte-add/v/2.1.0?activeTab=code

`@svelte-add/core` is _pinned_ to `2.5.2`, so it should **always** be fetching and using `@svelte-add/core@2.5.2`. Not sure if there could be any other explanation for it

**Comment by benmccann** [comment]
I can't reproduce the need for `@latest` with `npx` anymore. I first moved my `~/.npm` directory to a new location, did `npx svelte-add@2.0.1`, and then did `npx svelte-add` and that worked. Then I restored my `~/.npm` directory and did `npx svelte-add` and that surprisingly worked. Maybe it was something to do with `svelte-add` being a peer dependency of itself that's now been fixed in `svelte-add` version 2.2.0. I'm going to keep using it without `@latest` for awhile and see if it ever pops up again.
